packages/fxa-settings/src/pages/Signin/index.stories.tsx

```diff
@@ -19,12 +19,11 @@ const SigninWithProvider = ({
   email,
   isPasswordNeeded,
   serviceName,
-  ServiceLogo,
 }: SigninProps) => {
   return (
     <LocationProvider>
       <AppLayout>
-        <Signin {...{ email, isPasswordNeeded, ServiceLogo, serviceName }} />
+        <Signin {...{ email, isPasswordNeeded, serviceName }} />
```

Comment:
I took `ServiceLogo` out of here in favor of adding into the `serviceLogos` const in `CardHeader`; we can let that component manage what has a logo instead. Looked to me like it's only Pocket at the moment so I deleted a related story / test. If someone sees something to the contrary I can add it into this `serviceLogos` const and add the others back.

Reply:
Should adding in a logo for Pocket be optional? (e.g., `hasLogo` prop)? Wondering if all instances of the Pocket service name in CardHeaders should always be replaced by the logo or not 🤔

Reply:
I'm not 100% certain of the answer here, our service-mixin looks like this:
```
// Some clients have a custom logo in their service name
const SERVICE_LOGO = {
  '7377719276ad44ee': 'graphic-client-pocket', // pocket-mobile
  '749818d3f2e7857f': 'graphic-client-pocket', // pocket-web
};
...
const { service, subscriptionProductName, serviceName } = this.relier.pick(
  'service',
  ...

const serviceLogo = SERVICE_LOGO[service];
```

I think the question is if the "Pocket" `serviceName` covers both / only these client IDs. There is an additional "pocket-web" client I see in a `POCKET_CLIENTIDS` const, so we may want to target those IDs instead of the name. This will probably be a bit more clear once we have Dan's relier PR merged.

Regarding this + your comment [here](https://github.com/mozilla/fxa/pull/14844#discussion_r1099288618), I think we should leave a comment on one of the signin tickets with a link here. Perhaps FXA-6488 which is already bumped to a 5?

BTW, it looks like we're missing "You are now ready to use [serviceLogo]" in the existing `Ready` mustache file.

`ready.mustache`:
```
        {{#serviceLogo}}
            <!-- L10N: Pocket image follows `You are now ready to use` -->
            {{#unsafeTranslate}}You are now ready to use <div class="graphic %(serviceLogo)s">%(serviceName)s</div>{{/unsafeTranslate}}
        {{/serviceLogo}}
```

We'll want to account for that as well.

Reply:
Addressing the question about how to know if client is Pocket in [FXA-6488](https://mozilla-hub.atlassian.net/browse/FXA-6488) sounds good to me.

Ha, that `Ready` bit explains why I thought I'd seen the pocket logo in another spot then the `Signin` page but couldn't find it in fxa-settings. [FXA-6488](https://mozilla-hub.atlassian.net/browse/FXA-6124) might be a good time to address that missing phrase.